uplink/decorators.py

```diff
@@ -94,9 +94,11 @@ def modify_request(self, request_builder):
 
 # noinspection PyPep8Naming
 class headers(MethodAnnotation):
-    """
-    You can apply this decorator on a class to add headers to all of
-    its API definitions.
+    """Use as a decorator on a class to add Headers to all API calls.
+
+    Attributes:
+        _headers:  A dictionary containing header field as Key
```

Comment:
In the class docstrings, let's omit the attributes if they start with leading underscores, which indicates they should be treated as private methods.